.github/actions/hab-pkg-build-and-upload-linux/action.yaml

```diff
@@ -24,7 +24,7 @@ runs:
         HAB_AUTH_TOKEN: ${{ inputs.hab-auth-token }}
       run: |
         source results/last_build.env
-        sudo hab pkg install --auth $HAB_AUTH_TOKEN results/$pkg_artifact
+        sudo hab pkg install --auth $HAB_AUTH_TOKEN results/$pkg_artifact --channel LTS-2024
```

Comment:
I don't think this is going to do anything. Because it is installing from a local hart file, it won't pull from a channel.